docs/next.config.js

```diff
@@ -34,6 +34,16 @@ module.exports = withCSS({
     };
     return config;
   },
+  // Enable CORS headers in development to allow interacting with Snack
```

Comment:
@brentvatne, I think the CORS headers were added to allow InlineSnacks fetch the data it needs, right? If so, I don't think we need this anymore:

1. Locally, you _must use_ `http://localhost:3000` (instead of e.g. `0.0.0.0:3000`) because Chrome and other browsers now block HTTPS -> HTTP requests. The only URL with an exception for this is `localhost` 😄 
2. Because we are forced to use `localhost` now for cross HTTPS -> HTTP requests, these CORS headers are irrelevant. Because `localhost` is also handled differently from the CORS headers.

TL;DR; Looks like InlineSnacks works both with and without these headers on `localhost`, I don't think we need them anymore.

Reply:
nice, thanks for investigating this

Reply:
I'd suggest including a note about why it's safe to expose our content to any website:

```suggestion
  // Set CORS headers to enable cross-origin access in development to allow interacting with
  // Snack and for parity with production. The docs content is safe to expose to other domains as
  // it is entirely static.
```

Reply:
To confirm, we don't need these CORS headers anymore in `next.config.js`. All requests between HTTP and HTTPS sites are blocked by default, except for `localhost`. If you want to use a proxy, like ngrok, you can use the HTTPS without these CORS headers. I'll remove it from the config!

I also wonder if we need this in S3, I'll put it on my todo list to check with Nick.